packagedb/api.py

```diff
@@ -710,6 +710,101 @@ class PackageSetViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = PackageSetAPISerializer
 
 
+class PurlValidateViewSet(viewsets.ViewSet):
+    """
+    Take a `purl` and check whether it's valid PackageURL or not.  
+    Optionally set `check_existence` to true to check whether the package exists in real world. 
+    
+    **Note:** As of now `check_existence` only supports `apache`, `composer`, `deb`, `gem`, 
+    `github`, `golang`, `maven`, `npm`, `nuget`and `pypi` ecosystems.
+
+    **Input example:**
+
+            {
+                "purl": "pkg:npm/foobar@12.3.1",
+                "check_existence": true,
+            }
+    
+    Response contains:
+
+    - valid
+        - True, if input PURL is a valid PackageURL.
+    - exists
+        - True, if input PURL exists in real world and `check_existence` flag is enabled.
+    """
+    def get_view_name(self):
```

Comment:
I am just curious how `get_view_name` is used

Reply:
`get_view_name` is used for overriding the default view name.